custom/icds_reports/utils/aggregation_helpers/monolith/aww_incentive.py

```diff
@@ -84,7 +84,12 @@ def aggregation_query(self):
         /* update visits for cf cases (not in agg_ccs_record) */
         UPDATE "{tablename}" perf
         SET expected_visits = expected_visits + cf_data.expected,
-            valid_visits = valid_visits + cf_data.valid
+            valid_visits = valid_visits + cf_data.valid,
+            visit_denominator = round(expected_visits + cf_data.expected),
+            incentive_eligible = (ROUND(wer_weighed / GREATEST(wer_eligible, 1)::NUMERIC, 4) >= 0.6 
```

Comment:
Same thing here. This has some complex rounding in the query above and it gets rounded again in hte update

Reply:
Same as above, any rounded values are recalculated from scratch. It is why this does not refer to the visit denominator column, but instead uses the raw values that compose it.